--- a/chrome/browser/ui/ui_features.cc
+++ b/chrome/browser/ui/ui_features.cc
@@ -284,7 +284,7 @@
              base::FEATURE_DISABLED_BY_DEFAULT);
 #endif
 
-BASE_FEATURE(kSidePanelFlyoverAnimation, base::FEATURE_DISABLED_BY_DEFAULT);
+BASE_FEATURE(kSidePanelFlyoverAnimation, base::FEATURE_ENABLED_BY_DEFAULT);
 
 bool UseSidePanelFlyoverAnimation() {
   return base::FeatureList::IsEnabled(kSidePanelFlyoverAnimation);
